### DISCUSSION TITLE
Talk not Working After Manual Upgrade to NC 30.0.0 RC1

### DISCUSSION THREAD

**Discussion by frake65** [nextcloud/all-in-one#5130] [Questions] [answered]
Hello,
as I'm always an early adaptor (caused by my curiosity) I already upgraded my Nextcloud AIO 9.4.1 installations with Nextcloud 30.0.0 RC1.
Afterwards trying to have a call by Nextcloud Talk I get the message "The configured signaling server needs to be updated to be compatible with this version of Talk. Please contact your administration." - I'm using the standard solution with integrated Talk container.
I suppose, there could be an updated Talk (Spreed) container available already. Are you planning to integrate this within the next release of AIO?

**Comment by szaimen** [comment]
Hi, we are using the upstream image: https://github.com/nextcloud/all-in-one/blob/db55a1c2822eee4dd295883726f36f85fb292e3a/Containers/talk/Dockerfile#L4
As soon as that gets updated, we can also update it here :)

**Comment by szaimen** [comment, marked as answer]
spreed-signaling got updated to v2.0.0 with v9.5.0 beta. See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
